app/controllers/menus_controller.rb

```diff
@@ -17,8 +21,24 @@ def create
     end
   end
 
+  def edit
+    @menu = Menu.find(params[:id])
+    @dishes = Dish.all #this needs to be restricted to only dishes created by the restaurant later on
+  end
+
+  def update
+    @menu = Menu.find(params[:id])
+    @menu.update(menu_params)
+    # params[:menu][:dish_ids].each do |dish|
+    #   @menu.dish_ids << dish
+    #   @menu.save
+    # end
+    # binding.pry
```

Comment:
Sorry about that, was rushing to push this out before George walked in the door.